engine/statefulset.go

```diff
@@ -22,7 +22,7 @@ func checkRunningStatefulsetsConformity(ctx context.Context, l zerolog.Logger, s
 				return hasBeenPatched, err
 			}
 
-			l.Info().Str("statefulset", ss.Name).Msgf("scaling %s from 0 to %d replicas", ss.Name, repl)
+			l.Info().Str("statefulset", ss.Name).Msgf("scaling from 0 to %d replicas", repl)
```

Comment:
:question: why remove the names of the resources in the log message? This would have allowed us to grab all the information needed in one field (message) without having to filter on a key that varies `statefulset` sometimes, `deployment` sometimes...

the original goal of having a `Str()` field with the resource type and the resource name is for logs aggregator (loki...), in order to be able to filter only on a special kind of resource. But as a human operator, I would love to have to filter only on message to get all the stuff I need to debug (`| jq .message`).

Reply:
OK, it looked like duplicated work to me but I understand now.

Reply:
🤔 I thought about it a second time, and I still think it is duplicate information and effort.

* Most of the time while debugging as an operator, you do `grep` on the whole line, so you have the namespace and the message, even when formated as Json
* I re-read every log statement and we always add `... namespace '%s'", n.Name)`
  * This is prone to errors
  * A duplicate effort
  * An inconvenient because the namespace name is not located at the same place in the message

Reply:
I haven't used the `--human` flag that disables JSON logging yet, be maybe we will need to focus on it later to add the namespace name automatically like we do on JSON formated logs.